### ISSUE TITLE
Add support for Yotogi Class Skill vol. 3

### ISSUE THREAD

**Issue by Raimoo** [MainVirtualizer/CM3D2Editor#5]
Can you add support for Yotogi Class Skill vol. 3?

**Comment by kingwilly71** [comment]
Made a quick look over the code...
This could be done only after ver1.17 is supported, because the save file format changed.

**Comment by MainVirtualizer** [comment]
Forgot to close this, this is already done in commit 0e12e4d
